docs/dev-local-node.md

```diff
@@ -51,7 +51,7 @@ We are going to use environment variables for convenience, ENDPOINT_HTTP and END
     export ETHEREUM_PRIVATE_KEY="fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a"
 
     # Use dp to deploy to the configuration specified in your local environment (above)
-    npm explore augur.js -- npm run deploy:environment
+    npx dp deploy
```

Comment:
Should we replace `npm explore augur.js -- npm run deploy:environment` above with this?

Reply:
this is actually good, tested it.